packages/runtime/src/helpers/dev.ts

```diff
@@ -20,21 +23,31 @@ export const onPreDev: OnPreBuild = async ({ constants, netlifyConfig }) => {
   })
   await writeDevEdgeFunction(constants)
 
-  if (!existsSync(resolve(base, 'middleware.ts')) && !existsSync(resolve(base, 'middleware.js'))) {
-    console.log(
-      "No middleware found. If you did intend to use middleware, create a 'middleware.ts' or 'middleware.js' file in your project root to add custom middleware.",
```

Comment:
This was really just for my benefit when I was developing it, and I meant to remove it.